app/services/bwr_renderer.py

```
@@ -1,19 +1,20 @@
-from app.services.font_utils import FONTS_DIR
+from app.services.circuit_metadata import COUNTRY_MAP
 from app.services.renderer import Renderer
-from app.services.renderer_common import draw_results_header, load_track_image_asset
-from app.services.spectra6_renderer import TRACKS_DIR, Spectra6Renderer, logger
-from app.services.track_assets import build_track_stem_candidates
+from app.services.renderer_assets import prepare_color_track_image
+from app.services.renderer_theme import make_color_theme
+from app.services.spectra6_renderer import IMAGES_DIR, Spectra6Renderer
 from app.utils.bmp import encode_indexed_bmp_4bit, map_to_bwr_palette
 
+__all__ = ["BwrColors", "BwrRenderer", "COUNTRY_MAP"]
```

Ruff (0.15.21)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)